azure-devops/projects/pagopa-packages-projects/danger-custom-rules.tf

```
@@ -0,0 +1,160 @@
+variable "danger-custom-rules" {
+  default = {
+    repository = {
+      organization   = "pagopa"
+      name           = "danger-custom-rules"
```

```suggestion
      name           = "danger-custom-rules"
```
wrong repo name?
https://github.com/pagopa/danger-plugin-digitalcitizenship